tests/Avalonia.Controls.UnitTests/ComboBoxTests.cs

```diff
@@ -21,6 +23,7 @@ public class ComboBoxTests
         [Fact]
         public void Clicking_On_Control_Toggles_IsDropDownOpen()
         {
+            using var app = UnitTestApplication.Start(new TestServices(threadingInterface: Mock.Of<IPlatformThreadingInterface>(x => x.CurrentThreadIsLoopThread == true)));
```

Comment:
Why do you need these changes? Can it be "true" by default in the UnitTestApplication? Or create a simple implementation which would check current thread if it's the same as one where app was created?